packages/kbn-optimizer/limits.yml

```diff
@@ -143,7 +143,7 @@ pageLoadAssetSize:
   searchprofiler: 67080
   security: 81771
   securitySolution: 98429
-  securitySolutionEss: 16573
+  securitySolutionEss: 31781
```

Comment:
The new components added to this package are lazy loaded, but a package increase was still required.